core/src/main/java/io/questdb/griffin/SqlCompilerImpl.java

```diff
@@ -2523,7 +2524,7 @@ private void sqlShow(SqlExecutionContext executionContext) throws SqlException {
             // show server_version
             RecordCursorFactory factory = null;
             if (isTablesKeyword(tok)) {
-                factory = new TableListRecordCursorFactory();
+                factory = new TableListCursorFactory(configuration, AllTablesFunctionFactory.METADATA, AllTablesFunctionFactory.SIGNATURE);
```

Comment:
this will conflict with my PR https://github.com/questdb/questdb/pull/3972

two static args are not needed as parameters. Neither should be configuration. Engine is available from execution context